File: URI/1 - INICIANTE/Python/1150 - UltrapassandoZ.py
x = int(input())
y = 0
cont = 1
while y == 0:
    z = int(input())
    if(z > x):
        y = 1
y = 0
w = x
while y == 0:
    if(x > z):
        y = 1
    else:
        x += w
        w += 1
        cont +=1
print(cont)

File: URI/1 - INICIANTE/C/1150 - UltrapassandoZ.c
#include <stdio.h>

int main()
{
	int x, z, y,w, cont = 1;
	scanf(" %d", &x);
	w = 0;
	do{
		scanf(" %d", &z);
		if(z > x)
		{
			w = 1;
		}
	}while(w == 0);
	w = 0;
	y = x;
	do{
		if(x > z){
			w = 1;
		}else{
			x += y;
			y += 1;
			cont +=1;
		}
		
	}while(w == 0);
	printf("%d\n", cont);
	return 0;
}
